config/locales/en.yml

```diff
@@ -288,6 +288,7 @@ en:
     your_feedback: Your feedback
     dates_and_deadlines: Dates and deadlines
     organisation_permissions: How organisation permissions affect what users can do
+    application_guidance: How applications work
```

Comment:
Yes it should, I've made a comment to change the H1 to 'The teacher training application process' - so the title should match that. (I've removed the cycle date as we mention it in the first line and I think this will be better for Google if the H1 isn't changing year on year.